andela/hc-silicon
# #157694756 Job running too often

#### What does this PR do?
- Add an icon on the dashboard to show a job that is run too often
- Sends an email to the user to notify that a job is running too often

#### Description of Task to be completed?
When a job runs too often a user can see an icon on the dashboard and also receive an email notification each time a job run earlier than expected(too often).
#### How should this be manually tested?
- Run the application and click the check link to create a new check
- Run the job too often to receive a notification email and view an icon on the dashboard
#### What are the relevant pivotal tracker stories?
[Job running too often](https://www.pivotaltracker.com/story/show/157694756)

Screenshots
A sample of an icon on the dashboard

![image](https://user-images.githubusercontent.com/22454909/41131097-8c44541e-6ac2-11e8-81c8-f53fd06be81c.png)

Sample of the email a user receive

![image](https://user-images.githubusercontent.com/22454909/41131111-9b98e83a-6ac2-11e8-8b51-f8c3fdd97a85.png)
